### DISCUSSION TITLE
How to sell Chia on Monday

### DISCUSSION THREAD

**Discussion by berlef80** [Chia-Network/chia-blockchain#3275] [Q&A]
According to the Chia website transactions are supposed to be enabled on Monday at 10am PDT.  Does anyone know how to sell them?  What crypto exchange platforms will support it?  Will a price point be set when enabled or is it purely demand driven?
Thanks!

**Comment by bertybassett** [comment]
not on here, sorry you won't get answer.

**Reply by berlef80** [reply]
Confused - Why not and where would I find it?  What's the point of farming if selling them is secret?

**Reply by bertybassett** [reply]
because its illegal for chia to be seen giving advise on trading.  Just join up with the top 10 exchanges and you'll be covered.  THIS IS NOT TRADING ADVICE.

**Reply by berlef80** [reply]
Oh Sorry I didn't realize that saying what exchanges it would be available to sell on was considered trading advise.
